src/components/Chat/Chat.module.css

```diff
@@ -0,0 +1,58 @@
+.chatRoom {
+    font-size: 1em;
+    display: flex;
+    flex-grow: 1;
+    justify-content: center;
+    flex-direction: column;
+    align-items: stretch;
+    background-color: white;
+    border: 2px solid black;
+    border-radius: 1em;
+    margin: 0em 0.5em 0.5em 0.5em;
+    letter-spacing: 0.2em;
+}
+
+.chatBox {
+    font-size: 1em;
+    display: flex;
+    flex-grow: 1;
+    flex-direction: column;
+    max-height: 35vh;
```

Comment:
In the other pr I thought you got it to scroll properly without fixing the value?